R/Common_functions.R

```diff
@@ -143,12 +143,12 @@ commonFG <- list(
             stop(paste("An object with name", group_name, "does not exist"))
         }
     },
-    ls=function(recursive=FALSE, detailed=FALSE, index_type=h5const$H5_INDEX_NAME, order=h5const$H5_ITER_NATIVE, link_access_pl=h5const$H5P_DEFAULT,
-        dataset_access_pl=h5const$H5P_DEFAULT, type_access_pl=h5const$H5P_DEFAULT) {
+    ls=function(recursive=FALSE, detailed=FALSE, index_type=h5const$H5_INDEX_NAME, order=h5const$H5_ITER_NATIVE, link_access_pl=h5const$H5P_DEFAULT$id,
+        dataset_access_pl=h5const$H5P_DEFAULT$id, type_access_pl=h5const$H5P_DEFAULT$id) {
```

Comment:
Hi Mario,

wasn't sure about this change. It breaks with how all other functions are using property lists. These are currently implemented as R6 classes as property lists need to be cleaned at the end of life. 

Is there a particular reason why you needed this change?

Thanks